Makefile

```diff
@@ -7,6 +7,8 @@
 # configuration options that may be passed as environment variables:
 DEMO ?= DEMO
 GITHUB_USER ?= anonymous
+REANA_COMPONENT_PREFIX ?= reana
```

Comment:
Do we actually need a new variable? Shall we think of harmonising Python venv name, Minikube profile name, and Helm release name?  We could then have `reana`, or `reana-dev`, or `reana-nightlybuilds` used for everything... Since `Makefile` is used mostly by developers for developers, I guess this should do.